autoload/import_cost.vim

```diff
@@ -36,7 +36,9 @@ function! s:OnEvent(type, payload)
       call import_cost#virtual_text#Clear()
     endif
 
-    call s:EchoError(a:payload)
+    if !g:import_cost_silent
```

Comment:
It'll probably better to move this if statement to the `s:EchoError` function. That way, the `g:import_cost_silent` option will be global to all errors that this plugin throws.

Reply:
Hi, I moved this into `s:EchoError` .